### DISCUSSION TITLE
Contribute to docs.pwpush.com

### DISCUSSION THREAD

**Discussion by MindTooth** [pglombardo/PasswordPusher#2815] [General]
I might be stupid, but I can't find anywhere how I can contribute to the docs. 😅 

Specifically in https://docs.pwpush.com/docs/installation/#postgres-backend where `-e` needs to come before the image name to take effect.

Example:

```
docker run -d -p "5100:5100" pglombardo/pwpush:latest -e DATABASE_URL=mysql2://pwpush_user:pwpush_passwd@mysql:3306/pwpush_db

...

DATABASE_URL not specified. Assuming ephemeral backend. Database may be lost on container restart.
To set a database backend refer to https://docs.pwpush.com/docs/how-to-universal/#how-does-it-work
```

and 

```
docker run -d -p "5100:5100" -e DATABASE_URL=mysql2://pwpush_user:pwpush_passwd@mysql:3306/pwpush_db pglombardo/pwpush:latest 

...

According to DATABASE_URL database backend is set to mysql2:...
```

**Comment by pglombardo** [comment]
Hey @MindTooth - it's not you.  It was a new repo recently and I haven't opened it up yet.

Thanks for pointing that out!  I'll make that change and see if I can open the repo soon.  Need to write a README...

**Reply by pglombardo** [reply]
Fixed: https://docs.pwpush.com/docs/installation/#postgres-backend

Thanks @MindTooth!
